roles/jumbotron/tasks/fakta.yml

```diff
@@ -0,0 +1,31 @@
+---
+- name: Install our fakta service
+  copy:
+    dest: '/etc/systemd/system/fakta.service'
+    src: fakta.service
+    owner: root
+    group: root
+    mode: 0644
+
+- name: Install our fakta timer
+  copy:
+    dest: '/etc/systemd/system/fakta.timer'
+    src: fakta.timer
+    owner: root
+    group: root
+    mode: 0644
```

Comment:
Installing services and timers needs a call to `systemctl daemon-reload`. You can see one way to fix it here:
https://github.com/esmil/labitat-ansible/commit/b79071389517ed6d2360459283b9f41c503603a6#diff-8e2838d6d1d3ef550d10c166a7273fd6